### PR TITLE
chore(placement): add scrimba discount

### PR DESCRIPTION
**Pull request by LeoMcA**
## Summary

https://mozilla-hub.atlassian.net/browse/MP-1482

**Do not merge** before September 12, and deploy in sync with Scrimba.

---

## How did you test this change?

localhost:3000

### CODE REVIEW

File: client/src/ui/organisms/placement/index.tsx [hunk 1]
```diff
@@ -92,7 +92,22 @@ export function SidePlacement() {
 function TopPlacementFallbackContent() {
   const gleanClick = useGleanClick();
 
-  return (
+  return Date.now() < Date.parse("2024-10-12") ? (
```
Comment (caugner) [inline client/src/ui/organisms/placement/index.tsx:95]: Maybe inline the result?
```suggestion
  return Date.now() < 1728691200000 ? (
```

File: client/src/ui/organisms/placement/index.tsx [hunk 2]
```diff
@@ -92,7 +92,22 @@ export function SidePlacement() {
...
+    <p className="fallback-copy">
+      Learn front-end development with a 30% discount on{" "}
```
Comment (caugner) [inline client/src/ui/organisms/placement/index.tsx:97]: Double-check copy

File: client/src/ui/organisms/placement/index.tsx [hunk 3]
```diff
@@ -92,7 +92,22 @@ export function SidePlacement() {
...
+        href="https://scrimba.com/learn/frontend?via=mdn"
+        target="_blank"
+        rel="noreferrer"
+        onClick={() => {
+          gleanClick(BANNER_SCRIMBA_CLICK);
+        }}
+      >
+        Scrimba
+      </a>{" "}
+      - Limited time offer!
```
Comment (caugner) [inline client/src/ui/organisms/placement/index.tsx:108]: Especially this part

File: client/src/ui/organisms/placement/index.tsx (hunk 1 above)
Comment (LeoMcA) [reply client/src/ui/organisms/placement/index.tsx:95]: I prefer the more readable version: I'd hope date parsing is pretty well optimised in JS engines

File: client/src/ui/organisms/placement/index.tsx (hunk 3 above)
Comment (LeoMcA) [reply client/src/ui/organisms/placement/index.tsx]: checked and updated, ty

**Pull request merged**